lms/models.py

```diff
@@ -1,5 +1,7 @@
 from django.db import models
 
+from config import settings
```

Comment:
Правильнее настройки импортировать так
```suggestion
from django.conf import settings
```
иначе потом в тестах не сможешь их переопределять (иногда это необходимо)